### ISSUE TITLE
Unable select text on Internet Explorer 11

### ISSUE THREAD

**Issue by tmoreno** [readium/readium-shared-js#333]
#### Observed behaviour
When I highlight some text and go to another chapter I can't select book text anymore and write in input fields like textarea

#### Steps to reproduce
1. Open this epub with Internet Explorer 11: http://readium.github.io/readium-js-viewer/?epub=http%3A%2F%2Freadium.github.io%2Freadium-js-viewer%2Fepub_content%2Femerald-city-oz&
2. Go to chapter 1
3. Select some text.
4. Click on highlight text button
5. Go to other chapter
6. Try to do a text selection

### Product
* Readium cloud reader app

**Comment by rkwright** [comment]
@tmoreno This sounds like it might be a bug, but you have not provided enough information.  What Readium code base are you using?  The CloudReader?  Are you cloning from develop or master?  Or somewhere else.  What is the goal.  Readium doesn't preserve or retain any highlight information.  It's more of an experiment than anything else.

**Comment by tmoreno** [comment]
I'm using the CloudReader hosted on http://readium.github.io/readium-js-viewer and with the book "The Emerald City of Oz"

But I have the same bug with a build of the readium-js project from this commit: 665bdf0d90d5e0ae209b81eaa264293056cbf028


![screenshot](https://cloud.githubusercontent.com/assets/1009817/18903659/9d7b03b2-855a-11e6-946a-070d9a1d8541.png)

**Comment by danielweck** [comment]
The official deployement URL for the cloud reader (latest build from the develop branch). is:
https://readium.firebaseapp.com/?epub=https%3A%2F%2Freadium.github.io%2Freadium-js-viewer%2Fepub_content%2Femerald-city-oz

Can you please try again with this link?

**Comment by tmoreno** [comment]
The button for highlights doesn't appear in that cloud reader version, I can't highlight text.

**Comment by danielweck** [comment]
Ah, excellent point :)
(the plugin is disabled in the master branch / release, but we can reactivate it in the develop builds)

**Comment by danielweck** [comment]
Please try the above URL again once Travis CI has finished building Readium:
https://travis-ci.org/readium/readium-js-viewer/builds/163437854
Thanks :)

**Comment by danielweck** [comment]
I tried myself in Edge and IE. Works "fine", except for the mouse hover error in the console:
`Unable to get property 'annotationHoverIn' of undefined or null reference`
Definitely something to investigate, @JCCR

**Comment by tmoreno** [comment]
Yes, that the point related with my other issue: https://github.com/readium/readium-shared-js/issues/332 I can't remove created highlights.

In this one, I can select for highlight again when I change the chapter.

**Comment by tmoreno** [comment]
Hi,

Any good news?

**Comment by JCCR** [comment]
@tmoreno 
I ran across this issue as well. Let me see if someone in my team managed to find a solution.
